bench/bench_test.go

```diff
@@ -0,0 +1,69 @@
+package bench
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math/rand"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/tetratelabs/wazero/wasi"
+	"github.com/tetratelabs/wazero/wasm"
+	"github.com/tetratelabs/wazero/wasm/naivevm"
+	"github.com/tetratelabs/wazero/wasm/wazeroir"
+)
+
+func BenchmarkEngines(b *testing.B) {
+	buf, _ := os.ReadFile("case/case.wasm")
+	mod, _ := wasm.DecodeModule((buf))
```

Comment:
another bench we can consider is decoding and instantiating a module, as this could affect startup time

Reply:
DecodeModule doesn't depend on engines, but we should measure `store.Instantiate` where we actually compile functions using engines. Let's do later.